extensions/amp-web-push/0.1/validator-amp-web-push.protoascii

```diff
@@ -0,0 +1,42 @@
+#
```

Comment:
We've moved validator protoasciis up one level, outside of the version directory. E.g. `extensions/amp-web-push/validator-amp-web-push.protoascii`.